src/main/resources/static/api.yml

```diff
@@ -223,6 +223,8 @@ components:
           type: string
         alternateName:
           type: string
+        identifier:
```

Comment:
do we need to make this `required`?

Reply:
I think a later commit makes this required.

Reply:
oops, sorry, you're right!